hack/install-requirements.sh

```diff
@@ -20,10 +20,10 @@ go get -u "gopkg.in/gobuffalo/packr.v1/v2/packr2"
 go get -u "gopkg.in/onsi/ginkgo.v1/ginkgo"
 go get -u "gopkg.in/golang/mock.v1/mockgen"
 go get -u "golang.org/x/lint/golint"
-curl -s "https://raw.githubusercontent.com/helm/helm/master/scripts/get" | bash
+curl -s "https://raw.githubusercontent.com/helm/helm/v2.13.1/scripts/get" | bash -s -- --version 'v2.13.1'
```

Comment:
Can you please rebase so that this hack change is not included? #61 with the fix is merged.